### PR TITLE
Final merge of my edits

### PR DESCRIPTION
**Pull request by dallenmidd**
This looks great! I have just a couple little edits. According to the word count tool this is just under 3000. Feel free to reject my edits if you want or we can chat about them tomorrow.

### CODE REVIEW

File: paper/forestecology.Rmd [hunk 1]
```diff
@@ -17,7 +17,7 @@ abstract:
...
+  | 4. The package features i) `tidyverse`-like structure whereby verb-named functions can be modularly "piped" in sequence, ii) functions with standardized inputs/outputs of simple features `sf` package class, and iii) an S3 object-oriented implementation of the Bayesian linear regression model. These three facts allow for clear articulation of all the steps in the sequence of analysis and easy wrangling and visualization of the geospatial data. Furthermore, while the package only has Bayesian linear regression implemented, the package was designed with extensibility to other methods in mind.
```
Comment (dallenmidd) [inline paper/forestecology.Rmd:20]: "Forestry" has a sort of specific meaning that I don't think we want. To me it means study of logging or timber operations.

File: paper/forestecology.Rmd [hunk 2]
```diff
@@ -458,7 +459,7 @@ The resulting RMSE of `r model_rmse_cv` computed using cross-validation is `r if
...
+`run_cv()` also accepts the `run_shuffle` argument in order to permute competitor species identity as described in Section \ref{permutation-test}. Figure \ref{fig:scbi-simulation} compares model performance for 49 permutations of competitor species and RMSE calculations, both with and without cross-validation. Without cross-validation, competitor species identity does matter as the observed RMSE was significantly lower than the permutation null distribution of RMSE. However, once we incorporate spatial cross-validation, this improvement disappears. These results suggest that in this 9 ha subplot of the SCBI plot, competitive interactions do not depend on the identity of the competitor, which is the opposite of what has been observed in other locations [@allen_permutation_2020 @uriarte_spatially_2004]. This provides a striking example of the importance of cross-validation, as without it the over-fit model gives rise to an incorrect conclusion.
```
Comment (dallenmidd) [inline paper/forestecology.Rmd:462]: HEre I just really wanted to emphasize how cool this is! The CV changes the result. If you don't CV you are too confident in an overfit model that tells you that competitor identity matters. But this is not true when CVing!

Comment (rudeboybert) [reply paper/forestecology.Rmd:462]: Dang, you're right. I was so lost in the details of getting this paper out I didn't even realize this.

File: paper/forestecology.Rmd [hunk 3]
```diff
@@ -467,7 +468,7 @@ include_graphics("simulation_results/2021-03-03_scbi_49_shuffles.png")
...
+The `forestecology` package provides an accessible way to fit and test models of neighborhood competition. While the package is designed with ForestGEO plot data in mind, we envision that it can be modified to work on any single large, mapped forest plot in which at least two measurements of each individual have been taken. Furthermore, we also envision the package being used more generally for inventory plot-structure with many spatially separated plots, e.g. the US Forest Service Forest Inventory and Analysis plots [@smith_forest_2002]. In future versions of `forestecology` we also hope to include models that account for tree mortality in addition to tree growth. The package follows the `tidy` data design principles, leverages the `sf` package for spatial data, and S3 open-oriented model implementation structure. We hope that the package will increase the use of neighborhood competition models to better understand what structures plant competition.
```
[the remark this answers is not in the document]
Comment (rudeboybert) [reply paper/forestecology.Rmd:471]: Ah my bad. My slash and burn of the word count killed the nuance of the message

**Pull request merged**